src/components/MultiHopTrade/components/MultiHopTradeConfirm/hooks/useTradeExecution.tsx

```diff
@@ -50,6 +53,12 @@ export const useTradeExecution = (hopIndex: number) => {
     return cancelPollingRef.current
   }, [])
 
+  // The intermediary buy asset may not actually be supported. If it doesnt exist in the asset slice
+  // the it must be unsupported.
```

Comment:
because OCD
```suggestion
  // The intermediary buy asset may not actually be supported. If it doesn't exist in the asset slice
  // then it must be unsupported.
```

Reply:
Also: 🧠